docs/sources/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -42,6 +42,9 @@ manually install
 the [Custom Resource Definitions (CRDs)](https://github.com/grafana/agent/tree/main/production/operator/crds) for the
 Agent operator.
 
+If you prefer not to use the agent operator, then deploy a standalone Grafana Agent with the configuration
+provided in [Collect metrics and logs via Grafana Agent](#collect-metrics-and-logs-via-grafana-agent).
```

Comment:
The user should skip the following 'Credentials' and 'Helm chart values' sections if they don't want to use the operator right? As in, 'if you prefer not to use the agent operator', you should just skip down in the docs all the way to [Collect metrics and logs via Grafana Agent](#collect-metrics-and-logs-via-grafana-agent) right? 

If so, I'll suggest a tweak to the language. Right now I'm left wondering if I should deploy a standalone Grafana agent and then come back up to this point and complete the next two sections. 

I also think we could add something in here that says 'However, we recommend using the operator because it ensures you have all the right labels, etc'. (Basically we echo your argument to Vanguard for why they should use this approach instead of just using an agent they manage themselves).